import-export-cli/integration/testutils/devFirst_testUtils.go

```diff
@@ -246,5 +255,6 @@ func ValidateAPIWithImageIsExported(t *testing.T, args *InitTestArgs, DevFirstDe
 		base.RemoveDir(args.InitFlag)
 		base.RemoveDir(exportedPath)
 		base.RemoveDir(relativePath)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing a directory and nothing related to indexing is taking place in APIM side.